crmd/lrm.c

```diff
@@ -2077,6 +2077,19 @@ do_lrm_rsc_op(lrm_state_t * lrm_state, lrmd_rsc_info_t * rsc, const char *operat
         }
     }
 
+    if (!is_remote_lrmd_ra(NULL, NULL, rsc->id)) {
+        xmlNode *versioned_params = find_xml_node(msg, XML_TAG_VER_ATTRS, FALSE);
+
+        if (versioned_params) {
+            char *versioned_params_text = dump_xml_unformatted(versioned_params);
+
+            if (versioned_params_text) {
+                params = lrmd_key_value_add(params, XML_TAG_VER_ATTRS, versioned_params_text);
```

Comment:
I may be a little paranoid, but let's use "#" XML_TAG_VER_ATTRS for the magic attribute, so we don't limit what RA writers can use for their option names. (C string literal concatenation will make that "#versioned_attributes".)